tests/testthat/_snaps/ard_categorical.md

```diff
@@ -106,39 +106,39 @@
          group1         group1_level variable variable_level     context stat_name
```

Comment:
Since we're updating these snapshots, can we also add the `withr::local_options(list(width = 120))` within the testing chunk so the tibble prints without splitting into two sections?

Reply:
good call, done!